### DISCUSSION TITLE
제2회 콘테스트 개최 - 오디 캐릭터 편

### DISCUSSION THREAD

**Discussion by eun-byeol** [woowacourse-teams/2024-ody#257] [Ideas]
3가지 화면에 들어갈 캐릭터 그려오기!
보상 : 수다날 밥 (내돈ㅜㅜ)

1. 약속 참여 완료 화면
![image](https://github.com/user-attachments/assets/239d9012-bf63-496c-b0e5-7ac1d6941408)

2. 약속 개설 완료 화면
![image](https://github.com/user-attachments/assets/19b34207-6445-4151-a2bf-e8770da49ca5)

3. 약속 없는 경우 보여지는 화면
![image](https://github.com/user-attachments/assets/7affcf00-f522-403b-9a32-fa829dd8083d)

**Comment by aprilgom** [comment]
![오디 컨ㅂ](https://github.com/user-attachments/assets/14b8af05-4206-4f4c-ba02-52a363b10b0e)

**Reply by haeum808** [reply]
캬

**Reply by eun-byeol** [reply]
진짜 너무 귀엽다 진짜 너무 귀엽다 진짜 너무 귀엽다

**Comment by coli-geonwoo** [comment]
![001](https://github.com/user-attachments/assets/8ebc28a6-4225-4143-8b62-85dbfc0d7f3b)
![002](https://github.com/user-attachments/assets/8f89d706-f528-42bd-9804-87dfcbfe4451)
![003](https://github.com/user-attachments/assets/dc97e0b6-60c1-4625-86b4-c8c154bcb8c8)

![004](https://github.com/user-attachments/assets/69458b77-7ff5-483a-9607-065367138ea7)
-abe9-45dd-b210-ac5bf08c5f16)
![005](https://github.com/user-attachments/assets/e07c343f-27d2-40e9-8e0f-1a40e64278f3)

![006](https://github.com/user-attachments/assets/90974b14-a3cb-4409-a176-bdf621786968)
ser-attachments/assets/5d06357c-8dfc-49a4-bc27-fbc75f6643d6)

**Reply by kimhm0728** [reply]
ㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋ

**Reply by haeum808** [reply]
우왕

**Reply by aprilgom** [reply]
너무 서럽잖아요,..

**Reply by eun-byeol** [reply]
눈물 킹받아;-;ㅋㅋㅋㅋㅋㅋ

**Comment by mzeong** [comment]
<img width="692" alt="스크린샷 2024-08-05 오후 5 06 30" src="https://github.com/user-attachments/assets/c0f1d7d1-85d2-4caf-b38c-cbc5f33df24e">

**Reply by aprilgom** [reply]
근본

**Reply by haeum808** [reply]
원조

**Comment by eun-byeol** [comment]
### 표정
![IMG_0600](https://github.com/user-attachments/assets/7d88828c-e57c-49e1-8474-76d4ad1ae963)
![IMG_0601](https://github.com/user-attachments/assets/04a9d2be-f49f-4f91-bc60-321fc4978bee)
![IMG_0602](https://github.com/user-attachments/assets/3a80f3c2-bf70-491f-877a-8940011733e1)
![IMG_0603](https://github.com/user-attachments/assets/5a9adf13-c9c0-43b9-869c-f13a9577122a)
![IMG_0604](https://github.com/user-attachments/assets/2ec353ec-f4f6-4aed-bb29-bff12034e64a)
![IMG_0605](https://github.com/user-attachments/assets/e15f7cc3-5ed3-4c37-8357-1325351c07ea)
![IMG_0606](https://github.com/user-attachments/assets/02fd7941-adc9-4bf5-8d30-30c1a7511984)
![IMG_0607](https://github.com/user-attachments/assets/c25bc6df-1226-4ff6-9d55-3fcb0748cfa4)
![IMG_0608](https://github.com/user-attachments/assets/19b864fe-21d8-4b3a-b59c-3d5603a4939a)
![IMG_0609](https://github.com/user-attachments/assets/7110cd69-a6ee-4f42-a32f-57bf907bd808)

### 달력
![IMG_0613](https://github.com/user-attachments/assets/fa071329-ded6-44a1-817a-2d2f2a440412)
![IMG_0614](https://github.com/user-attachments/assets/ea288d54-d327-4f18-9704-08d730708b0f)

### 덤
![IMG_0610](https://github.com/user-attachments/assets/32fd66d3-745c-4338-b68d-080cb59f167e)
![IMG_0612](https://github.com/user-attachments/assets/91d874a8-42f4-4906-af65-ce8379102f36)
![IMG_0616](https://github.com/user-attachments/assets/02ae53c6-a74a-49de-b7b3-d4688bedf6fc)

**Reply by haeum808** [reply]
오옹

**Reply by kimhm0728** [reply]
스티커로 만들고싶음 개귀여움

**Reply by kimhm0728** [reply]
아 재능충 에바지

**Comment by kimhm0728** [comment]
<img width="785" alt="스크린샷 2024-08-05 오후 5 05 57" src="https://github.com/user-attachments/assets/9d2b5494-8e99-43e0-99ba-6de598100ec2">

아 큰일이다 ㄷㄷ

**Reply by haeum808** [reply]
오오

**Comment by haeum808** [comment]
![이름 없는 노트북](https://github.com/user-attachments/assets/deb23619-7f5d-418c-8616-5186bd216937)

**Reply by kimhm0728** [reply]
야채부락리다

**Reply by aprilgom** [reply]
쿵야 + 메이플 몹

**Comment by haeum808** [comment]
![제목_없는_아트워크-1](https://github.com/user-attachments/assets/b62788c6-b07c-414b-aeb9-8db91f48a4bf)
![제목_없는_아트워크-1 2](https://github.com/user-attachments/assets/e68ecc1a-136e-4089-aa78-57430d7872bf)
![제목_없는_아트워크-1 3](https://github.com/user-attachments/assets/cada5120-84d6-44cb-b845-ae86f01325ef)

**Reply by aprilgom** [reply]
귀로 하트 하는거 너무 킹받는데요

**Reply by haeum808** [reply]
반전술식
